jetty-ee11/jetty-ee11-servlet/src/main/java/org/eclipse/jetty/ee11/servlet/ErrorHandler.java

```diff
@@ -73,6 +73,14 @@ public boolean errorPageForMethod(String method)
         };
     }
 
+    private static Object getRequestErrorAttribute(HttpServletRequest request, String servletApiName, String jettyName)
```

Comment:
Why is this method necessary?   
I thought the policy was that we would always set the jetty(core) attribute and then any request wrapper used for an error dispatch to a filter/servlet would intercept the servlet attributes and lookup the jetty ones

So our code should then only ever need to lookup jetty(core) attributes?  Unless there is a case where an application sets them and we are expected to act on them?

Reply:
That is part of the refactoring effort, which is a much wider scope than just satisfying the tck.